### DISCUSSION TITLE
Wrong cities in response

### DISCUSSION THREAD

**Discussion by mLR5** [gravitational/teleport#33910] [Q&A]
Hi guys, 

I'm new using the Teleport API so excuse me in advance if my question makes no sense but when I try to request the city of Paris by searching with the names, I get cities from Jamaica in my response. [https://api.teleport.org/api/cities/?search=Paris](url)
Here is an example of what I get in return  {'_links': {'city:item': {'href': 'https://api.teleport.org/api/cities/geonameid:3489854/'}},
    'matching_alternate_names': [],
    'matching_full_name': 'Kingston, Kingston, Jamaica'},

I don't understand why I have these type of cities, can anyone help me understand what's going on ? 

Thank you

**Comment by webvictim** [comment]
Hi - this discussion forum relates to questions for https://goteleport.com, not https://teleport.org :)
